scripts/myrial

```diff
@@ -143,6 +146,12 @@ def main(args):
                 raise "Options dot and --plan are incompatible"
 
             print operator_to_dot(processor.get_logical_plan())
+        elif opt.opt_dot:
+            if opt.from_repr:
```

Comment:
I don't understand. Are you saying that we should be able to use `myrial -d --plan`?